myriad-scheduler/src/main/java/org/apache/myriad/api/ClustersResource.java

```diff
@@ -1,27 +1,30 @@
 /**
- * Copyright 2015 PayPal
- *
- * Licensed under the Apache License, Version 2.0 (the "License"); you may not
- * use this file except in compliance with the License. You may obtain a copy of
- * the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
- * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
- * License for the specific language governing permissions and limitations under
- * the License.
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ * 
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ *   Unless required by applicable law or agreed to in writing,
+ *   software distributed under the License is distributed on an
+ *   "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *   KIND, either express or implied.  See the License for the
+ *   specific language governing permissions and limitations
+ *   under the License.
  */
-package com.ebay.myriad.api;
+package org.apache.myriad.api;
 
 import com.codahale.metrics.annotation.Timed;
-import com.ebay.myriad.api.model.FlexDownClusterRequest;
-import com.ebay.myriad.api.model.FlexUpClusterRequest;
-import com.ebay.myriad.configuration.MyriadConfiguration;
-import com.ebay.myriad.scheduler.MyriadOperations;
-import com.ebay.myriad.scheduler.NMProfileManager;
-import com.ebay.myriad.state.SchedulerState;
+import org.apache.myriad.api.model.FlexDownClusterRequest;
+import org.apache.myriad.api.model.FlexUpClusterRequest;
+import org.apache.myriad.configuration.MyriadConfiguration;
+import org.apache.myriad.scheduler.MyriadOperations;
+import org.apache.myriad.scheduler.NMProfileManager;
+import org.apache.myriad.state.SchedulerState;
 import com.google.common.base.Preconditions;
```

Comment:
Now you've got all these org.apache's between two com.*s
Can you do a pass fixing the alpha-order?